src/components/draggable/draggable.pw.tsx

```diff
@@ -0,0 +1,123 @@
+import { expect, test } from "@playwright/experimental-ct-react17";
+import React from "react";
+import {
+  draggableItem,
+  draggableItemByPosition,
+} from "../../../playwright/components/draggable";
+import { checkAccessibility } from "../../../playwright/support/helper";
+
+import {
+  DraggableCustom,
+  DraggableDefault,
+  DraggableDifferentContainers,
+} from "../../../src/components/draggable/components.test-pw";
+
+test.describe("Check functionality for Draggable component", () => {
+  ([
+    ["One", 1],
+    ["One", 2],
+    ["One", 3],
+    ["One", 4],
+    ["Two", 3],
+    ["Three", 1],
+    ["Four", 2],
+  ] as const).forEach(([record, destinationId]) => {
+    test(`should drag ${record} and re-order DraggableItem to the ${destinationId} position`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DraggableCustom />);
+
+      const draggableItemElementRecord = await draggableItem(page, record);
+      const draggableItemByPositionElementDestinationId = await draggableItemByPosition(
+        page,
+        destinationId
+      );
+      await draggableItemElementRecord.dragTo(
+        draggableItemByPositionElementDestinationId
+      );
+
+      await expect(draggableItemByPositionElementDestinationId).toContainText(
+        record
+      );
+    });
+  });
+
+  test("should not change the order of the DraggableItems when dragged outside of the DraggableContainer", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DraggableDifferentContainers />);
+
+    const draggableItemElementFive = await draggableItem(page, "Five");
```

Comment:
The suggestion regarding unnecessary 'awaits' when declaring constants that I included in the first test applies to this and all subsequent tests.